### PR TITLE
Developer documentation for Job Submission

### PR DESCRIPTION
**Pull request by rexissimus**

### CODE REVIEW

**Comment by remram44** [comment]
Exact duplicate of #1127?

**Comment by rexissimus** [comment]
Thanks. I had not committed my changes yet. This is based on top of #1127 but I wanted to put it in a separate pull-request.

File: doc/usersguide/job_submission.rst [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+* View standard output/error for running jobs - If implemented by the handle.
+* Delete running workflows/modules.
+* Set time interval for automatic job checking.
+* Set flag for waiting for job to finish.
+
+.. _fig-list_job-monitor:
+
+.. figure:: figures/job_submission/job_monitor.png
+   :align: center
+   :width: 7in
```
Comment (remram44) [inline doc/usersguide/job_submission.rst]: These figures are missing

Comment (rexissimus) [inline doc/usersguide/job_submission.rst]: Thanks, fixed with 00837ee3e17632310d48cfab73726772832842f8.

**Comment by rexissimus** [comment]
The missing myjob package makes the usersguide vtl link test fail. I will fix it by reordering the tests so that the workflow that installs the package is executed first.

**Comment by rexissimus** [comment]
@remram44, we could get the package from a link, but it is also needed by the tests, so I did not think it was necessary to fetch a file that will always be present anyway?

**Comment by remram44** [comment]
Oh you copy it from the local bundled one! Got it.

**Comment by rexissimus** [comment]
The tests have bee fixed by c72b8067062c80babd2f24d2fb0957506862e37f.

**Pull request merged**